auditorium/src/views/components/auditorium/embed-code.js

```diff
@@ -49,7 +55,12 @@ const EmbedCode = (props) => {
     )
   }
 
+
   const snippet = `<script async src="${window.location.origin}/script.js" data-account-id="${model.account.accountId}"></script>`
+  const snippetWithSRI = `<script async src="${window.location.origin}/script.js" data-account-id="${model.account.accountId}" integrity="sha256-${process.env.SCRIPT_INTEGRITY_HASH}"></script>`
+  const [useSnippetWithSRI, setActive] = useState(false);
+  let embeddedSnipped = useSnippetWithSRI ? snippetWithSRI : snippet;
+  let buttonText = useSnippetWithSRI ? 'Hide integrity hash' : 'Show with integrity hash';
```

Comment:
As Offen is translated into multiple languages, all copy needs to be localized properly. Doing this is relatively easy: a global `__` function is available in all modules that you can wrap a string in, marking as "to be localized":

```js
__('Hide integrity hash')
```